en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -48,15 +51,15 @@ Connect to the flight control GPS interface using a cable.
 
 ## Safety Switch
 
-The dedicated safety switch that comes with the V5+ is only required if you are not using the recommended Neo GPS (which has an inbuilt safety switch).
+The dedicated safety switch that comes with the V5+ is only required if you are not using the recommended *Neo v2 GPS* (which has an inbuilt safety switch).
 
-If you are flying without the GPS you must attach the switch directly to the GPS1 port in order to be able to arm the vehicle and fly (If you use the old 6-pin GPS, please read the definition of the bottom interface to change the line).
+If you are flying without the GPS you must attach the switch directly to the `GPS1` port in order to be able to arm the vehicle and fly (If you use the old 6-pin GPS, please read the definition of the bottom interface to change the line).
 
 
 ## Buzzer
 
-If you do not use the recommended GPS the buzzer may not work.
-You may need to add an external buzzer instead.
+If you do not use the recommended *Neo v2 GPS* the buzzer may not work.
+<!-- You may need to add an external buzzer instead. -->
```

Comment:
@cuhome FYI If you're not going to explain how to add an external buzzer we shouldn't state that they should do this. I have therefore commented this (in both docs)

Reply:
You can delete it